sidebars.js

```diff
@@ -221,7 +221,11 @@ module.exports = {
         },
       ],
       Optimize: [
-        "components/optimize/what-is-optimize",
+        {
+          type: "link",
+          label: "What is Optimize?",
+          href: "/optimize/what-is-optimize",
+        },
```

Comment:
This is the other end of the foreseen down-side to the multi-version docs -- [having to rewrite the sidebars files, and duplicate them, due to the two plugins not knowing about each others' docs](https://github.com/camunda/camunda-platform-docs/pull/906#discussion_r874097861).